iot_api/user_api/events/alert_events.py

```diff
@@ -18,6 +18,8 @@
 from iot_api.user_api.models.NotificationPreferences import NotificationPreferences
 from iot_api.user_api.models.NotificationDataCollectorSettings import NotificationDataCollectorSettings
 from iot_api.user_api.models.NotificationAlertSettings import NotificationAlertSettings
+from iot_api.user_api.models.NotificationAssetImportance import NotificationAssetImportance
```

Comment:
These models could be imported directly from models module to remove some clutter.

Reply:
Done